nextcord/guild.py

```diff
@@ -2621,14 +2628,14 @@ async def create_role(
             fields['name'] = name
 
         if icon is not MISSING:
-            if icon is None:
-                fields['icon'] = icon
-            elif isinstance(icon, str):
+            if isinstance(icon, str):
```

Comment:
once again, this was never resolved, please add comments on why you believe this is resolved in this thread

Reply:
Now, this is resolved because we are now using utils._obj_to_base64_data which allows passing `None` explicitly to the icon kwarg.
```py
        if icon is not MISSING:
            if isinstance(icon, str):
                fields['unicode_emoji'] = icon
            else:
                fields['icon'] = await utils._obj_to_base64_data(icon)
```